migrations/versions/571d8a2570a6_.py

```diff
@@ -0,0 +1,63 @@
+"""empty message
+
+Revision ID: 571d8a2570a6
+Revises: 66897499754f
+Create Date: 2019-09-20 09:25:39.672266
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "571d8a2570a6"
+down_revision = "66897499754f"
+branch_labels = None
+depends_on = None
+
+job_statuses = sa.Enum("started", "completed", "errored", name="job_statuses")
+
+assay_upload_status = sa.Enum(
+    "started",
+    "upload-completed",
+    "upload-failed",
+    "merge-completed",
+    "merge-failed",
+    name="assay_upload_status",
+)
+
+
+def upgrade():
+    assay_upload_status.create(op.get_bind())
+
+    op.execute("ALTER TABLE assay_uploads ALTER COLUMN status TYPE text")
```

Comment:
From the model def I thought this column will be renamed. Shouldn't it be?

Reply:
I don't think so. `AssayUploads` still have a `status` field; this PR adds an additional `AssayUploads.status_details` field in the other migration (below).